### ISSUE TITLE
Consider moving the project to an organization namespace

### ISSUE THREAD

**Issue by baco** [DetachHead/basedpyright#370]
Please consider creating a base GitHub Organization for the project, and moving ownership of the project to said organization's namespace.

It happened in the past with similar projects, such as with [python-language-server](https://github.com/palantir/python-language-server), that lead to (either because the project was dropped by the original owner, or the owner lost their credentials, etc.) the project having to be forked.  Such circumstances hinder the community formed around those projects.

This can be easily avoided when having an organization namaspace with several administrators.  As this project is gaining more adoption among the community it's probably a good idea to offer back the security of being an organization maintained project.

This “bug report” should only be considered as a **suggestion** to improve on project's governance.  Current work on the project already has the necessary release frequency and maintenance from it's owner.

**Comment by DetachHead** [comment]
i have been considering creating an organization. though rest assured i have no plans to abandon this project any time soon, and i won't be adding any administrators that i don't fully trust (the recent [xz backdoor incident](https://en.wikipedia.org/wiki/XZ_Utils_backdoor) is a perfect example of why i'm skeptical to do so).

this is my first project that's gained any traction. i'm not an experienced maintainer so i want to make sure i do this correctly. i'd be interested to learn about how other repos went from someone's personal project to an organization. what was their process for picking trustworthy and reliable maintainers?

**Comment by baco** [comment]
> i have been considering creating an organization. though rest assured i have no plans to abandon this project any time soon, and i won't be adding any administrators that i don't fully trust (the recent [xz backdoor incident](https://en.wikipedia.org/wiki/XZ_Utils_backdoor) is a perfect example of why i'm skeptical to do so).

Fair enough, that's a really valid point.  Thanks for the reassurance.  And thanks for already considering forming an organization.

> what was their process for picking trustworthy and reliable maintainers?

I don't know for sure the process.  I haven't started an organization myself either 😕.  I guess that comes with time, through exhaustive and consistent code reviews showing trust in the collaborators' production from PRs.

_**Disclaimer** (to avoid misread intentions): I am not proposing myself to be part of the organization, since I don't know anything of Typescript or Javascript, which are core to the project._
